src/main/java/bio/terra/app/utils/startup/StartupInitializer.java

```diff
@@ -11,6 +15,20 @@ private StartupInitializer() {}
 
   public static void initialize(ApplicationContext applicationContext) {
     // Initialize jobService, stairway, migrate databases, perform recovery
-    applicationContext.getBean("jobService");
+    applicationContext.getBean(JobService.class);
+
+    // Initialize the Synapse DB if needed
+    Synapse config = applicationContext.getBean(AzureResourceConfiguration.class).getSynapse();
```

Comment:
The code here for initialize on startup isn't taking advantage of spring mechanisms. I think you can do this using `PostConstruct` on `AzureSynapsePdao`.

Reply:
Ooh, interesting...lemme look at changing this over